### PR TITLE
Adds Info struct and a bit of hlsp Parse

### PR DESCRIPTION
**Pull request by fmvilas**
### What?
Adds:

* `Info` struct.
* A way to extract specification extensions from any object.
* The latest AsyncAPI version.
* Some code for hlsp.Parse (don't take this too seriously, it's just a test)

### CODE REVIEW

File: models/info.go [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+	Title          string         `json:"title,omitempty"`
+	Description    string         `json:"description,omitempty"`
+	TermsOfService string         `json:"termsOfService,omitempty"`
+	Contact        *Contact       `json:"contact,omitempty"`
+	License        *License       `json:"license,omitempty"`
+	Version        string         `json:"version,omitempty"`
+}
+
+// Unmarshal unmarshals JSON
+func (value *Info) Unmarshal(data []byte) error {
```
Comment (rmelian) [inline models/info.go]: Is there a way we can create an interface for all models to implement Unmarshal and Marshal. I'm seen now these two methods all over the place doing the exact thing

Comment (fmvilas) [reply models/info.go]: Yes, would be great to avoid repetition.

**Comment by rmelian** [comment]
LGTM, I'll send a PR later if I find a way to reuse marshal and unmarshal methods

**Comment by fmvilas** [comment]
Merging to unblock @rmelian.

**Pull request merged**